app/grandchallenge/reader_studies/migrations/0008_auto_20210226_0949.py

```diff
@@ -1,17 +1,60 @@
-# Generated by Django 3.1.6 on 2021-02-24 15:03
+# Generated by Django 3.1.6 on 2021-02-26 09:49
 
 from django.db import migrations, models
 
 import grandchallenge.core.validators
 
 
+IMAGE_PORT_EMPTY_PLACEHOLDER = 9
+
+
+def remap_image_ports(apps, _):
```

Comment:
You cannot run a data migration in the same migration as schema migration in postgres. I strongly urge you not to change the field type here, feel free to increase the length of the field.

Reply:
> migration in postgres.

@MikeOverkamp-diag , fyi.

Reply:
I was afraid you would need a separate migration for this, but it somehow did seem to work when you tried it this way.